XenAdmin/TabPages/GeneralTabPage.cs

```diff
@@ -1329,6 +1346,16 @@ private void generateGeneralBox()
             s.AddEntry(FriendlyName("host.uuid"), GetUUID(xenObject));
         }
 
+        private string PoolExpireString()
+        {
+            if (licenseStatus.CurrentState == LicenseStatus.HostState.Expired)
+                return Messages.LICENSE_EXPIRY_TITLE;
+            else if (licenseStatus.CurrentState == LicenseStatus.HostState.Free)
+                return Messages.LICENSE_EXPIRED;

```

Comment:
The names of these strings are confusing. I think it's time to change them. 
- Rename LICENSE_EXPIRED to LICENSE_UNLICENSED
- Rename LICENSE_EXPIRY_TITLE to LICENSE_EXPIRED and use lowercase for "expired" ("License expired")
Please also make sure that we rename them in the localised resx files.